### PR TITLE
Add supported protocols to agent page

### PR DESCRIPTION
**Pull request by ksmith94**
This adds a section to the agent documentation about which protocols are currently supported. Related to #4293

### CODE REVIEW

File: packages/docs/docs/agent/index.md [hunk 1]
```diff
@@ -173,6 +173,12 @@ npm run agent <base_url> <client_id> <client_secret> <agent_id>
 
 Keep the terminal open so that you can see the logs. At this point you can start sending messages to the agent.
 
+## Supported Protocols
+
+- [HL7 v2](https://www.hl7.org): A low-level transfer protocol for HL7 content.
+- [DICOM C-Store](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html): A protocol used for storing medical imaging data.
```
Comment (rahul1) [inline packages/docs/docs/agent/index.md:179]:
```suggestion
- [DICOM C-Store](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html): A protocol used for storing metadata about medical images.
```

File: packages/docs/docs/agent/index.md [hunk 2]
```diff
@@ -173,6 +173,12 @@ npm run agent <base_url> <client_id> <client_secret> <agent_id>
 
 Keep the terminal open so that you can see the logs. At this point you can start sending messages to the agent.
 
+## Supported Protocols
+
+- [HL7 v2](https://www.hl7.org): A low-level transfer protocol for HL7 content.
+- [DICOM C-Store](https://dicom.nema.org/dicom/2013/output/chtml/part07/sect_9.3.html): A protocol used for storing metadata about medical images.
```
Comment (codyebberson) [inline packages/docs/docs/agent/index.md:179]: "DICOM" is the protocol

"C-STORE" (all caps) is one operation

We support both "C-STORE" and "C-ECHO" today.

If we wanted to put a salesy spin, we could say "other DICOM operations such as C-FIND, C-GET, and C-MOVE are coming soon"

File: packages/docs/docs/agent/index.md [hunk 3]
```diff
@@ -173,6 +173,12 @@ npm run agent <base_url> <client_id> <client_secret> <agent_id>
...
+- [ASTM](https://www.astm.org/): Used to transfer data between clinical instruments and computer systems.
```
Comment (codyebberson) [inline packages/docs/docs/agent/index.md:180]: We may want to flag this as "experimental" or "alpha"

Comment (ThatOneBro) [reply packages/docs/docs/agent/index.md:180]: Do we want to wait until we add it back to the agent before we list it as supported? Potential documentation drift

Comment (rahul1) [reply packages/docs/docs/agent/index.md:180]: @ThatOneBro - Is this dependent on the serialport change we made? I thought the protocol and the connection was independent. 

@ksmith94 - I support flagging as "alpha"

File: packages/docs/docs/agent/index.md (hunk 2 above)
Comment (rahul1) [reply packages/docs/docs/agent/index.md:179]: @ksmith94 can you update to clarify the distinction between the protocol and the operations?

File: packages/docs/docs/agent/index.md (hunk 3 above)
Comment (ksmith94) [reply packages/docs/docs/agent/index.md:180]: This has been flagged as "alpha". Let me know if you want to remove it or make any other changes.

**Review by rahul1 — APPROVED** [review]

**Pull request merged**